RELEASE.md

```diff
@@ -17,47 +17,49 @@ Steps for a HALE release
 ========================
 
 On release branch:
-
-1. Merge origin/master into release
+1. Merge origin/master into release and commit the changes
 2. Set the release version for application bundles and build
    `./updateversionnumbers.groovy --release <RELEASE-VERSION>`
 3. Create a commit
 4. Create a tag for the new release
 5. Push to origin/release and push tags
+6. Create a PR
 
 On master branch:
-
 1. Set new snapshot version for bundles and features
    `./updateversionnumbers.groovy -o <RELEASE-VERSION> -n <NEW-VERSION>`
 2. Set new snapshot version for application
    `./updateversionnumbers.groovy --snapshot <NEW-VERSION>`
+3. Create a PR   
 
 Example
 =======
 
 On release branch:
-
 ```
 git merge origin/master
-./updateversionnumbers.groovy --release 2.9.0
+./updateversionnumbers.groovy --release 5.1.0
 git add -A
-git commit -m "Updated application versions for 2.9.0 release"
-git tag 2.9.0
+git commit -m "Updated application versions for 5.1.0 release"
+git tag 5.1.0
 git push origin release
 git push --tags
 ```
 
 On master branch:
-
 ```
-./updateversionnumbers.groovy -o 2.9.0 -n 3.0.0
-./updateversionnumbers.groovy --snapshot 3.0.0
+./updateversionnumbers.groovy -o 5.2.0 -n 5.3.0
+./updateversionnumbers.groovy --snapshot 5.3.0
 ```
 
 After the release
 =================
-1. Create Windows installer.
-Prerequisites: install Wix Toolset v.3.11 [available here](https://github.com/wixtoolset/wix3/releases/tag/wix3112rtm)
-Then run `build.bat product -o windows -a x86_64 HALE` from the root directory.
-2. Draft a new release for the latest release at https://github.com/halestudio/hale/releases and publish it.
-3. Update [download page](https://github.com/wetransform/www.wetransform.to/blob/deploy/app/downloads/index.html) so that the latest builds are available for download from https://www.wetransform.to/downloads/
+1. Create Windows installer:
+   - Prerequisites: install Wix Toolset v.3.11 [available here](https://github.com/wixtoolset/wix3/releases/tag/wix3112rtm)
+   - While on the release branch, execute `build.bat product -o windows -a x86_64 HALE` from the root directory.
+2. Drafting a New Release on GitHub:
+	- Gather the files generated in Jenkins from the latest successful build of the release branch.
+	- Utilize these files to draft the release on the [HALE GitHub repository](https://github.com/halestudio/hale/releases), including the .msi file generated in step 1.
```

Comment:
The spelling `HALE` should not be used anymore, it's long been deprecated in favour of `hale` or `hale»studio`.